contract/Portkey.Contracts.CA/CAContract_Actions.cs

```diff
@@ -162,13 +198,70 @@ public override Empty ReportPreCrossChainSyncHolderInfo(ReportPreCrossChainSyncH
 
         return new Empty();
     }
+    
+    private bool CreateCaHolderInfoWithCaHashAndCreateChainIdForZkLogin(ManagerInfo managerInfo, GuardianInfo guardianApproved,
```

Comment:
`NoncePayload` needs to be checked again actual operation taken in current transaction, e.g. `AddManager` for `CreateCAHolder`.